spec/features/severed_relationships_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Severed relationships page' do
+  include ProfileStories
+
+  describe 'GET severed_relationships#index' do
+    let(:event) { Fabricate(:account_relationship_severance_event, account: bob.account) }
+
+    before do
+      as_a_logged_in_user
+
+      Fabricate.times(3, :severed_relationship, local_account: event.account, relationship_severance_event: event.relationship_severance_event)
```

Comment:
Do we need all three ... or is having one enough to get the thing we want on the page?

Reply:
One is enough, but also in an earlier version I had defined incorrect unique indexes.